csharp/src/Google.Protobuf/FieldCodec.cs

```diff
@@ -72,7 +72,7 @@ public static FieldCodec<ByteString> ForBytes(uint tag)
         /// <returns>A codec for the given tag.</returns>
         public static FieldCodec<bool> ForBool(uint tag)
         {
-            return new FieldCodec<bool>(input => input.ReadBool(), (output, value) => output.WriteBool(value), CodedOutputStream.ComputeBoolSize, tag);
+            return new FieldCodec<bool>(input => input.ReadBool(), (output, value) => output.WriteBool(value), 1, tag);
```

Comment:
is the ComputeBoolSize method still needed? Also, I think we should define a constant in https://github.com/protocolbuffers/protobuf/blob/d9ccd0c0e6bbda9bf4476088eeb46b02d7dcd327/csharp/src/Google.Protobuf/CodedOutputStream.ComputeSize.cs#L40 rather than hardcoding the size values here.

same below

Reply:
@jtattermusch - I've pushed changes based on feedback.